src/Generator/PhpstormGenerator.php

```diff
@@ -33,7 +33,9 @@ public function generate() {
 	protected function build(array $map) {
 		$overrides = [];
 		foreach ($map as $directive) {
-			$overrides[] = $directive->build();
+            if (is_object($directive) && method_exists($directive, "build")) {
```

Comment:
Docblock states 

    @param \IdeHelper\Generator\Directive\BaseDirective[] $map
    
Are you sure?
There seems to be more wrong then and we need to fix that instead of trying to only fix the symptom

Reply:
I get an array of Tasks from the Queue Plugin and own Tasks:

```
if (is_array($directive)) {
    debug($directive);
    continue;
}
``` 

![Bildschirmfoto 2022-05-05 um 16 41 08](https://user-images.githubusercontent.com/77938819/166948552-ba4d48d7-fcae-442d-80ed-5006b59d3308.png)

Reply:
If I find the time. Came here for some quick annotations ;)